crates/server/src/api/content.rs

```diff
@@ -23,25 +25,31 @@ pub struct ContentConfig {
 impl ContentConfig {
     pub fn build_router(self) -> Result<Router> {
         Ok(Router::new()
+            .route("/:content_id", get(get_content))
             .route("/", post(upload_content))
-            .fallback_service(get_service(ServeDir::new(&self.content_path)).handle_error(
```

Comment:
As discussed 1:1, this was an attempt to make the content URLs be more consistent with other services that use `DynHash`-based ids in their paths so that the client doesn't have to know to replace the `:` with `-`.

I will back out this change and we'll potentially solve the inconsistency another way in the future.